packages/shared/src/components/LazyImage.tsx

```diff
@@ -52,11 +54,6 @@ function LazyImageComponent(
     imageProps = { src, className: baseImageClass };
   } else if (asyncImageSupport) {
     imageProps = { src, loading: 'lazy', className: baseImageClass };
-  } else {
-    imageProps = {
-      className: `lazyload ${baseImageClass}`,
-      'data-src': src,
-    };
```

Comment:
Since 'asyncImageSupport' is a const set to true at the top of this file, this else statement wont ever run